### DISCUSSION TITLE
Access files via network directly?

### DISCUSSION THREAD

**Discussion by garland410** [rg35xx-cfw/rg35xx-cfw.github.io#218] [Q&A] [answered]
Is there a way to update files via a windows computer using FTP or some other network solution. I know there's syncthing but I don't really understand that and to be honest reading about it makes me scared of mishaps and lost data. 

I'd just like to access the card from my PC without having to remove it every time. Thanks.

**Comment by uzumak11** [comment]
Your device will be accessible over the network already. 

If on windows, go to network devices (as long as you are on the same network). Drive will be called BATOCERA

username: root
password: linux

**Comment by garland410** [comment, marked as answer]
Not showing up for some reason, despite being on same network and having internet access. Hmm... gonna have to keep digging...

Edit: So finally after many failures I managed to gain access using the WinSCP app after enabling something called SMB file sharing support in windows features (Though that may or may not be what finally allowed WinSCP to work) 

Haven't tried to copy anything yet but at least I got it to finally recognize and browse the device in some manner.

Edit 2:

Got it! Had to right click, my pc and tell it to map network drive, assign a letter, and setup the login credentials there.
